cfgov/unprocessed/js/modules/FlyoutMenu.js

```diff
@@ -53,11 +53,12 @@ function FlyoutMenu( element ) { // eslint-disable-line max-statements, no-inlin
   var _isExpanded = false;
   var _isAnimating = false;
 
-  var _expandTransition;
-  var _collapseTransition;
-  var _expandTransitionMethod;
+  var _expandTransition = null;
+  var _expandTransitionMethod = null;
   var _expandTransitionMethodArgs = [];
-  var _collapseTransitionMethod;
+
+  var _collapseTransition = null;
```

Comment:
So why not just use https://github.com/cfpb/cfgov-refresh/blob/1ee49f2425f279a56aa85ce817024a989de494d3/cfgov/unprocessed/js/modules/util/standard-type.js#L25?



> Hmm, undefined vars actually do get set to the global singleton https://jsfiddle.net/xLc6fkze/


You are talking about a primitive value and not a singleton. 

From the ECMAScript spec:

> 4.3.9 undefined value
> 
> primitive value used when a variable has not been assigned a value
> 
> 4.3.10 Undefined type
> 
> type whose sole value is the undefined value
> 
> 4.3.11 null value
> 
> primitive value that represents the intentional absence of any object value
> 
> 4.3.12 Null type
> 
> type whose sole value is the null value